### DISCUSSION TITLE
2024-05-02 - As per our conversation or Checked By - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by samsmithssw** [SSWConsulting/SSW.Rules.Content#8485] [Chewing the Fat]
Hey SSW'ers,

Let's talk about these two rules:
https://www.ssw.com.au/rules/as-per-our-conversation-emails/
and
main rule (where the comments are shown) https://www.ssw.com.au/rules/checked-by-xxx/

Common topics - food for thoughts:
- Are you sold on this rule and do you try to get it "Checked by" before you hit 'Send'?

- Some people add "as per our conversation" to indicate the email is authorized via a prior conversation. Did you ever get confused about the rule and add "as per our conversation" when you had not discussed all the contents in the email?
 
- Do you write an email with what you want to say before you get onto a call with your client/approver?

**Comment by Freego1783** [comment]
I was clearly not used to doing this before I came to work for SSW and I was kind of reluctant in the beggining but now I can see why it's a good practice! The "check-by" part is not always easy to do for us baguettes in France because of time zones though.

**Comment by tiagov8** [comment]
Loved the analogy of "checked by" being a "PR approval".

**Comment by AlexRothwell** [comment]
There was discussion in the video about "as per our conversation" not including new _topics_ which I agree with - be topics should be separate emails. However I think it's also useful when it sometimes contains new _information_ such as when you were going to quickly check for an answer to a question after the call. In those cases I'd still provide an overview of the fact that I was going to follow up on the question but also include the result

**Comment by drwharris** [comment]
I now think of them like a "checked by" on a for the record email. "Hey [customer], I am going to send this email to you to confirm our discussion but would like you to check it first"

**Comment by camillars** [comment]
Like a lot of people here, I was not used to these practices, especially the checked by. 

Being a non native English speaker, it really helped me when I first joined SSW in 2018 to improve my written communication skills and email clarity. These are super helpful rules 🤓

**Comment by ozdemir-mehmet** [comment]
I prefer "As discussed" to "As per our conversation" (too wordy), and yes use it often.

Checked By is of limited use with add-ons like Grammarly that fix spelling, grammar, and more importantly "tone". Combine that with LLMs, instances of getting an actual "Checked by" should be rare.

**Reply by MarcusIrmscherSSW** [reply]
I have always been a fan of good communication and the "As per our conversation" rule is a great tool and something I've just naturally done.  The "Check by" rule however has taken a little getting used to, it definitely comes in handy.

**Reply by jakebayliss** [reply]
"As per our conversation" seems very "Karen" to me

**Reply by jaydenalchin** [reply]
+1- "As discussed" feels a lot more friendly and I use _As per our convo_ a lot more than _Checked by_

**Reply by Calinator444** [reply]
I think checked by is about more than just spelling and punctuation though. If you get a checked by, you're often checking either that:
1. You understand the task at hand (assuming the person checking the email assigned the task)
2. The task your sending makes sense and is a good decision (assuming a third party is checking the email)

**Reply by bradystroud** [reply]
Grammar and spelling should be perfect before a 'Checked by'. 

IMO 'Checked by Mehmet' means Mehmet agrees and is across the tasks/decision (so ask him for help)
its more similar to a '+1'

**Comment by SamBWagner** [comment]
I like both of these concepts. I'm definitely one to lean on a quick call over the herculean effort of getting a million checked by's. But, when the content is important, I'll seldom skip this procedure.

**Comment by ncn-ssw** [comment]
- If you're recording a decision, you need to ensure that the decision maker agrees with the record of that decision.
- If correspondence is important, it should always be checked - it is too easy to read what you thought you wrote rather than what you did.

**Comment by sethdaily** [comment]
Yes yes and yes

**Comment by joshbermanssw** [comment]
Im a fan of both and I think there have been a few times a 'checked by' has saved me from sending v2's v3's etc.

**Comment by Tanya-Leahy** [comment]
Being new to the company, "checked by" has helped me a lot.  I have gained a better understanding of how things are done at SSW by getting others to do a "checked by" on my emails.

**Comment by ozair-ashfaque** [comment]
I also prefer to have the email checked by the person with whom I had the conversation with for "As per our conversation" email

**Comment by kikibianc** [comment]
This is one of those rules that are hard to learn, easy to master.
Difficult to get going, but when you get in the habit of doing it, it becomes easy - it's good to see the gaps in your thinking when it's being checked by someone else.

Also, don't forget the test please, which is just a bigger checked by/as per our conversation together :D 

- https://www.ssw.com.au/rules/request-a-test-please/
- https://www.ssw.com.au/rules/conduct-a-test-please/

**Comment by stephanfakossw** [comment]
Very important one to remember. Helps decision making and creates efficiencies by improving comms.
Yes, no , yes...

**Comment by leahy268** [comment]
I struggled with "Checked by" for a start but it certainly stops misunderstandings.  If I am able to get someone to give me a checked by then I know that they understand completely before I send the email

**Comment by alexleessw** [comment]
At first I didn't understand "checked by xxx" but now can see it's a great tool

**Comment by AntPolkanov** [comment]
These rules are the best among others!

Before joining SSW, I felt like I had no idea how to use emails properly 😄 Previously, I considered email as something obsolete and merely a means to transition conversations into instant messaging.

**Comment by TobyChurches** [comment]
'Checked by' is a great rule, especially when trying to navigate important issues where it is essential that the recipient fully understands the content. I also love and try to use 'As per our conversation' as much as possible, its nice having a clear paper trail on decisions and comments to refer back to (Usually to cover myself 😄).

**Comment by matt-goldman** [comment]
Like many of our rules, these are very SSW specific. There is a lot of value in these, especially in the concept of shared accountability in the "Checked by" rule. However if we want these rules to be general and relevant to people outside of SSW, we need to surface this kind of intent in the rules. Otherwise they just look silly.

We also need to be very aware of using "Checked by" in emails to clients. Nobody in the world except SSW does this so we need to be cognisant of how clients will interpret this. The first time I received an email from someone with "Checked by" (it was one of my welcome emails before I started) I thought the implication was that the person had their "training wheels" on. My concern here is that it can disinspire confidence in the person sending the email in the eyes of the client.

**Reply by christoment** [reply]
I never thought of that second point - having seen as the person on training is not good!

**Comment by christoment** [comment]
I'm thinking in day-to-day use case of this rule, and one particular use case stand out. "As per our conversation" is really useful especially when talking over the phone with e.g. customer service so you can keep track record of the conversation with the other person.

My friend is currently in trouble because he didn't have a record of a phone conversation, and the other person backed down from the statement. An "as per our conversation" email would definitely help in this scenario!

**Comment by YalalCakirSSW** [comment]
I like "checked by" a lot. It reduces the pressure on my shoulder, and both of the rules ensure clear communication.  😅

**Comment by amankumarrr** [comment]
It was challenging for me when I first joined SSW, but as time went on, I began to appreciate the process more. Having someone review my emails meant entrusting them with my content, which provided a valuable opportunity for learning and gaining a fresh perspective on my topic. 

As a developer, it's crucial to have our work checked by the Product Owner (PO). This ensures accountability, alignment with project requirements, and formal agreement on the delivered work.

**Comment by jakebayliss** [comment]
Love both rules, although since I send most of my important emails to clients I tend to not add 'Checked By' at the top. I think this is because at the beginning I assumed it made me look 'new'. However once trust is built with the client I am sure they will be okay and even understanding that you get important emails checked before sending

**Comment by duanxinhuan** [comment]
check & pass is always the best.
It would be even better we also need a check&cancel system and get it work as well :)

**Comment by AttackOnMorty** [comment]
It's important to give constructive feedback when checking emails.

**Comment by jimmy1992abc** [comment]
I quite like the rules, as long as they don't become a blocker to the task itself. I believe the purpose of the 'Checked by' or 'As per our conversation' rules is to reduce misunderstandings and maintain a good record, but they shouldn't serve as a quality check for every email sent. I encourage people to use the 'Checked by' rule as much as possible, but consider the costs and benefits. You shouldn't spend 15 minutes on a task that should ideally be done in 2 minutes and is less important.

**Comment by zacharykeeping** [comment]
These are definitely some good, solid rules to have. "Checked by" especially I think is important for avoiding easily avoidable mistakes.

**Comment by bettybondoc** [comment]
I use either rules often when sending important emails - e.g. sending new email tasks to someone, recording design approvals (esp Adam's), sending a for the record, etc. It was hard to get used to it in the beginning but it becomes some sort of a habit later on. And they do help minimise misunderstandings.

**Comment by steven0x51** [comment]
Love the rules, the "as per our conversation" rule reduce the chance of misunderstanding and the "checked by" rule is important for avoiding mistakes are sent out.

**Comment by jeoffreyfischer** [comment]
I have a terrible short-term memory so the "As per my conversation" is definitely of great help! I actually need to use it to ensure that important conversations are stored somewhere safe.
I also love the "Checked by" rule because none of us is perfect, so a second set of eye is always welcome to ensure the an email is clear and easily understandable.

**Comment by PennyWalker** [comment]
I'm a fan of 'as per our conversation' and checked by because it adds confidence that I understood everything correctly and a 'sanity check' that I'm doing things the best way. I'm always happy to check other people's emails too because I think it helps me learn and solidify the best practices also.

**Comment by Ben0189** [comment]
Both of these are my favorite rules, they help improve communication and prevent misunderstandings

**Comment by chrisschultzssw** [comment]
Both great rules, and I use them often.

I do think there's a place for emails like:

> As per our conversation, X
> Also Y 
> And also Z

Maybe these need to be more explicit, i.e.:

> As per our conversation, X
> After our conversation I found that Y 
> And also Z

**Comment by danielmackay** [comment]
As per our conversation and checked by are great rules to apply for important or sensitive emails.  However, not everything needs to be checked.

I see the medal rating slightly different as presented in the video.

- Bronze - As per our conversation
- Silver - Bronze + I tried to call
- Gold - Bronze + checked by

Great to see @drwharris in a video!

**Comment by uly1** [comment]
Both super important rules... love love love them!

**Comment by Dhruv-0987** [comment]
Was an awesome video and I am sold on the rules. It would always be a good thing if I am going into a discussion and I have an email checked by other people.

**Comment by synnoveisabel** [comment]
These are very good rules to have! "As per our conversation" helps documenting discussions and decisions so it doesn't get lost in memory. And having a "checked by" is very useful, especially when you're new to the company and still learning the ropes.

**Comment by tombui99** [comment]
Very important rule that I use almost everyday 🙂

**Comment by ricksu978** [comment]
Appreciate the culture we have at SSW, not just the habit of getting things checked, people also keen on offering the help to check things!

**Comment by JeanThirion** [comment]
I've always been doing "As per our conversation" emails - even way before SSW.
But the "checked by" was new... Oh god I hated this so much. And the indetation. And so many other rules.

... and now I'm struggling with emails not following our standards 🤣

**Reply by wicksipedia** [reply]
💯 - I've been checking Lisa's emails for her business.... and when I tell her to reformat for quoting and indentation....

![CleanShot 2024-05-03 at 10 56 21](https://github.com/SSWConsulting/SSW.Rules.Content/assets/600044/9f28c67e-048a-40a1-a25d-9c6d8e6ab6ca)

**Comment by bahjatalaadelssw** [comment]
Helps with clarity, expectations and accountability.

**Comment by tomek-i** [comment]
_&lt;sarcasm&gt;_
The most difficult part before SSW: working with legacy projects, difficult clients, world hunger
after SSW: writing a single email
_&lt;/sarcasm&gt;_

The as per our conversation email to record decisions OR getting a written confirmation (even a 👍in a PBI) is super important
to put accountability back to the approver.
The checked by is great to get a second set of eyes and to find out if the email is easy to understand and follow or needs some refactoring :-) 

Oh, boy do I have war stories to tell 😂 where people did not follow the `as per conversation` rule with clients

**Comment by MattParkerDev** [comment]
`As per our conversation` is very important.

**Comment by Hona** [comment]
Its good to know about 'checked via Teams Chat' etc.
Also, good to see examples in the rule that you can reword 'As per our conversation' and not be so rigid

**Comment by babakamyljanovssw** [comment]
Both rules are important. English as second language for me, "Checked by" rule helped a lot to correct the email before hitting to send.

**Comment by tkapa** [comment]
> Are you sold on this rule and do you try to get it "Checked by" before you hit 'Send'?

Yes - It's always easier to have the peace of mind that if someone forgets something down the line that they did check things beforehand.

> Some people add "as per our conversation" to indicate the email is authorized via a prior conversation. Did you ever get confused about the rule and add "as per our conversation" when you had not discussed all the contents in the email?

Not Really - Sometimes I play fast and loose with it, but I generally won't just make up the things that we talked about.

> Do you write an email with what you want to say before you get onto a call with your client/approver?

Most of the time yes - It's just easier to get your thoughts down so that you can stay on track and keep things concise.

**Comment by cvionnet** [comment]
The 'checked by' rule ensures that both the writer and the reviewer focus on the form and content. As mentioned in the video, this makes them both accountable. Additionally, both parties learn and improve because what one person has not yet learned may be what another has already forgotten 😁

**Comment by zl24622** [comment]
Use as per our conversation very often and think it's helpful. 
Checked by not too often, mainly because of the time spent. But if it's an important email (e.g. sent to a client), will get people checked before sending it.

**Comment by miichaelsmedley** [comment]
As per our conversation, it has saved me many times and checked by also, often picking up silly errors that as an author you don't see.

**Comment by Calinator444** [comment]
At first I thought it seemed redundant getting someone to check an email you're sending them. Really though an email isn't just for direct communication between you and the recipient. It's a receipt that confirms that a conversation took place, and, more often than not that a decision was made, or a task was assigned.

**Comment by tino-liu** [comment]
Combining "Checked by..." and "As per our conversation" really helps with clearing things up. This "receipt" is so important that if anything happens, there's always a point to fall back and refer to. Great rules!

**Comment by Marxoz** [comment]
I use both these on a daily basis. The "as per our conversation" is a must have for me. It also allows me to go back and look at what was last discussed on a project after a rough weekend or some time of :) 

The checked by is just as good. I won't always include it in the email at the top, it all depends on the client, but I still get it checked regardless.

**Comment by jimmidier** [comment]
Draft an email before calling the approver is a good practise to avoid misunderstandings originated from verbal communication

**Comment by Aibono1225** [comment]
Both rules are important for efficient communication. I also love SSW's culture where everyone is willingly helping to check through.

**Comment by awaltos** [comment]
I like getting a Checked By if I am not 100% sure that the content of my email is accurate. Even though there is some overhead, Checked By is a good way to verify intent and potentially avoid confusion. Sometimes a different set of eyes can reveal something that wasn't quite obvious at first pass. Internally, everyone knows the rule and knows what to expect with this process, but a client may be confused about this.
As per our conversation emails are great for having a record in history for both parties.

**Comment by Levijj22** [comment]
Super important in the sales world - everyone has a terrible memory.

**Comment by GordonBeeming** [comment]
I love the more conversational videos, I enjoy getting a 2nd set of eyes on some emails, it helps confirm I understand what I thought I understand too 😅

**Comment by arig2024** [comment]
Love the ‘as per our conversation’. 
‘checked by..’ is great - as long as the person checking can be obtained in a timely manner.

**Comment by vladislav-kir** [comment]
Both rules are extremely useful to ensure that everyone is on the same page about the work that needs to be done.
